gdb2cm.go

```diff
@@ -16,6 +16,7 @@ import (
 var (
 	dashboardFile = kingpin.Flag("file.dashboard", "Grafana dashboard JSON file to convert.").Short('f').Required().ExistingFile()
 	manifestFile  = kingpin.Flag("file.output", "Output file for the dashboard configmap.").Short('o').Default("").String()
+	compact       = kingpin.Flag("file.compact", "Output file with compact JSON embedded in ConfigMap.").Short('c').Bool()
```

Comment:
this should have a default of false.

Reply:
It defaults to false by default (haha).  I updated to explicitly have a default of "false"